packages/nextjs-mf/src/plugins/NextFederationPlugin/index.ts

```diff
@@ -179,6 +178,7 @@ export class NextFederationPlugin {
         ...defaultShared,
         ...this._options.shared,
       },
+      ...(isServer ? {} : { manifest: { filePath: '/static/chunks' } }),
```

Comment:
Why only client side. If you enable it in one, you should enable in the other otherwise if these apis are used server side, it will not have the right data for server to server manifests.

Reply:
I did not dig into it but somehow, the server manifest was being generated and in the right folder (static/ssr).

Since for my implementation I was only using the client-side, figured I shouldn't mess with the server one.

Reply:
I will come back to this and see how it's being generated on the server and see if I can set the right path here too.

Reply:
it should dump it into the static/ssr dir - if it generates a server manifest then enable it and just use the client one for your needs.

Reply:
Please update PR to enable ssr output as well for manifest

Reply:
hey @ScriptedAlchemy!
when you have the time, could you take a look into this again, please?